package.json

```diff
@@ -64,6 +66,7 @@
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-path-utils": "^1.0.0",
     "ember-cli-string-utils": "^1.1.0",
+    "ember-cli-typescript-blueprint-polyfill": "^0.1.0",
```

Comment:
Will update this to `^1.0.0` once we know there aren't any other changes needed.